modules/actor/sheet/actor-sheet.js

```diff
@@ -522,7 +522,8 @@ export default class ActorSheetWfrp4e extends ActorSheet {
                     if (testDuplicate.result.minormis || testDuplicate.result.majormis || testDuplicate.result.catastrophicmis) {
                       break;
                     }
-                    if (test.item.cn.SL == test.item.cn.value) {
+                    let spell = game.actors.get(test.speaker.actor).items.get(test.item)
+                    if (spell.cn.SL == spell.cn.value) {
```

Comment:
I don't really like this, why was this changed?

Reply:
Because when I tested it again, I had only itemid instead of actual item in the object. If you have better idea how to get the spell from its id, I would update it.